README.md

```diff
@@ -341,19 +380,25 @@ Create the instance config file as `instance_config.json` in the `/deploy` direc
 
 Copy `global_config.py` from the root directory into the `/deploy` directory and update all values as needed.
 
+```bash
+# Navigate to the deploy directory.
+cd deploy
+# Copy the global_config.py file into the deploy directory.
+cp ../global_config.py .
```

Comment:
why does global config need to be copied to deploy/? I found it a little frustrating to have to make changes to two files when I was trying to get it working. can the deploy code read from the global config in the root directory?